usefulstuff.c

```diff
@@ -650,6 +650,28 @@ int xdebug_format_output_filename(char **filename, char *format, char *script_na
 					}
 				}	break;
 
+				case 'K': { /* XDEBUG_FILE in cookie */
+					zval **data;
+					char *char_ptr, *strval;
+					char *sess_name;
+					
+					sess_name = "XDEBUG_FILE";
+					
+					if (PG(http_globals)[TRACK_VARS_COOKIE] &&
+						zend_hash_find(Z_ARRVAL_P(PG(http_globals)[TRACK_VARS_COOKIE]), sess_name, strlen(sess_name) + 1, (void **) &data) == SUCCESS &&
```

Comment:
I prefer sizeof(sess_name) here, and probably I'd prefer that line 656 and 658 are the same.

I would also argue that the name should not be XDEBUG_FILE, but instead XDEBUG_EXTRA_INFO.

What would be even better if it was an #define somewhere as a "constant".